shenyu-admin/pom.xml

```diff
@@ -173,8 +173,8 @@
         </dependency>
 
         <dependency>
-            <groupId>mysql</groupId>
-            <artifactId>mysql-connector-java</artifactId>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
```

Comment:
mysql 8.3.0 , modify group id and artifactId?

Reply:
yes, mysql-connector-java driver`s  latest version is 8.0.33, which still has vulnerability. new driver modified group id and artifactId